examples/source/1.components/amp-script/index.html

```diff
@@ -1,5 +1,5 @@
 <!--
-  An [amp-script](content/amp.dev/documentation/components/reference/amp-script.md) hello world example.
+  An [amp-script](content/amp-dev/documentation/components/reference/amp-script.md) hello world example.
```

Comment:
shouldn't this have a leading slash as well?

Reply:
Indeed! Fixed.